### PR TITLE
update for Go 1.12

### PR DESCRIPTION
**Pull request by go101**

### CODE REVIEW

**Comment by go101** [comment]
Since Go 1.12, [the print order of map entries becomes constant](https://golang.org/doc/go1.12#fmt), though their iteration order is still randomized.

File: examples/maps/maps.sh [hunk 1]
```diff
@@ -6,4 +6,6 @@ v1:  7
 len: 2
 map: map[k1:7]
 prs: false
-map: map[foo:1 bar:2]
+map: map[bar:2 foo:1]
+
```
Comment (eliben) [inline examples/maps/maps.sh]: Remove two extra newlines

**Comment by eliben** [comment]
> Since Go 1.12, [the print order of map entries becomes constant](https://golang.org/doc/go1.12#fmt), though their iteration order is still randomized.

You're right, I had forgotten about that. The PR LGTM with one minor comment to fix

**Comment by mmcgrana** [comment]
Great thanks. I'll merge and fix + build on master.

**Pull request merged**